Source/Core/Core/HW/WiimoteEmu/Camera.cpp

```diff
@@ -105,6 +106,12 @@ void CameraLogic::Update(const Common::Matrix44& transform)
   std::array<CameraPoint, leds.size()> camera_points;
 
   std::transform(leds.begin(), leds.end(), camera_points.begin(), [&](const Vec3& v) {
+    if (!(IOS::gpio_out & IOS::GPIO::SENSOR_BAR))
```

Comment:
You could skip the std::transform entirely if the sensor bar is off. Just `camera_points.fill(INVISIBLE_POINT)`.

Reply:
Done.  I did it this way initially because I didn't want a massive diff from the indentation change when I was initially experiencing, but that doesn't really matter.